libultraship/libultraship/GlobalCtx2.cpp

```diff
@@ -82,4 +83,56 @@ namespace Ship {
             std::cout << "Log initialization failed: " << ex.what() << std::endl;
         }
     }
+
+    std::fstream GetSaveFile(std::shared_ptr<ConfigFile> config, std::ios_base::openmode additionalFlags = 0) {
```

Comment:
Some of this code is OOT specific. We should move most of it back to OOT rather than libultraship. libultraship should only expose getting and writing raw bytes from a save file.